contrib/gce/configure.sh

```diff
@@ -126,8 +126,16 @@ else
 
   # TODO(random-liu): Put version into the metadata instead of
   # deciding it in cloud init. This may cause issue to reboot test.
-  version=$(curl -f --ipv4 --retry 6 --retry-delay 3 --silent --show-error \
-    https://storage.googleapis.com/${deploy_path}/latest)
+  if [ $(arch) == "aarch64" ]; then
```

Comment:
Is it better to use `uname -m`? `arch` isn't on all linux systems

Reply:
Ah good point. Fixed.